ewc-affiliate/install-validator-centos-7-production-nethermind.sh

```diff
@@ -0,0 +1,493 @@
+#!/bin/bash
+
+# Make the script exit on any error
+set -e
+set -o errexit
+DEBIAN_FRONTEND=noninteractive
+
+# Configuration Block - Docker checksums are the image Id
+export NETHERMIND_VERSION="nethermind/nethermind:1.8.89"
+NETHERMIND_CHKSUM="sha256:6f92d584e6188932a0683948447e2d14824418e3a3443ca7d30ea7c29309bfaa"
+
+export NETHERMINDTELEMETRY_VERSION="latest"
```

Comment:
Please set a specific version. We have to deliver only stable versions to clients. 
With `latest` we're not sure what will be installed. 

Most of our clients are not technical. It's hard to get from them which version they have, so we want to minimalize "unknown" stuff.

Please do this change for all installation scripts

Reply:
ok, done